connect-integrations.md

```diff
@@ -0,0 +1,194 @@
+# Tributary: Consuming Urban Airship Connect on Behalf of a Customer
+
+> A tributary is a stream or river that flows into a larger
+> stream or main stem (or parent) river or a lake. A tributary does not flow
+> directly into a sea or ocean. Tributaries and the main stem river drain the
+> surrounding drainage basin of its surface water and groundwater, leading the
+> water out into an ocean.
+
+Urban Airship Connect is a streaming HTTP API which provides near-real-time
+mobile event data. The goal of the project is to give customers access to 
+the data we collect on their behalf. However, many customers don't want to
+spend the resources building a bespoke pipeline from our API to their analytics 
+engine. Enter Tributary, which takes the stream of Connect mobile data into 
+the ocean of third-party analytics and data storage tools.
+
+## Overview
+
+Tributary is a distributed system that can scale arbitrarily with customer
+demand. It's functionality is split across hardware nodes on our internal VPN, 
+and AWS hosted services. It relies particularly heavily on Amazon Web Services
+Elastic Cloud Compute Service (AWS ECS), which essentially ensures that the docker
+containers you'd like to be running are. This doesn't guarantee that your
+processes are running: ECS can fail to find a node that has enough resources to
+fulfill your specification, or configuration errors might prevent the process
+inside the docker container from ever doing any work, but it does mean that you
+can just shutdown the process in case of exception, and be confident that it
+will get started back up again. 
```

Comment:
Are there particular annoying problems that are avoided by this or another strategy that this is superior to?